File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.cs
public class Solution {
    public int RemoveDuplicates(int[] nums) {
        int numLen = nums.Length;
        if (numLen <= 1) {return numLen;}
        int lastIndex = 1;
        for(int i = 1; i < numLen; i++) {
            if(nums[i] != nums[i - 1]) {
                nums[lastIndex] = nums[i];
                lastIndex++;
            }
        }
        return lastIndex;
    }
}

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        set<int> numSet;
        for(int num : nums){
            numSet.insert(num);
        }
        int i = 0;
        for (auto it = numSet.begin(); it != numSet.end(); it++){
            nums[i] = *it;
            i++;
        }
        return numSet.size();
    }
};